### ISSUE TITLE
Failed to send message (403;

### ISSUE THREAD

**Issue by varazir** [sm00th/bitlbee-discord#181]
Hello, 

just installed this plugin and connected to the server. Now when I say anything in the channel I get this error in the main bible channel 

Failed to send message (403; <message> ) 

Running on a ubuntu server with irssi

**Comment by sm00th** [comment]
Please read through "Debugging" section in README and try to get a debug.log (make sure you are not posting tokens/passwords). 403 is HTTP's "Forbidden" code so are you sure you are properly connected and have the rights to post to the channel you are trying to send the message to?

**Comment by varazir** [comment]
Don't wan't to restart bitlbee use it to much other things well some other day. I know you need the extra debugging data. I can post in the channel using the web app.

**Comment by sm00th** [comment]
I think this might be the same as #182, so if the latest commits fix sending messages for you, please close this issue.

**Comment by varazir** [comment]
@sm00th 

Just did  git pull and compiled still getting 

23:43 <@root> discord - Error: Failed to get backlog (403). ( **This is fine** )
23:43 <@root> discord - Error: Failed to send message (403; Hello).

**Comment by dgw** [comment]
@varazir Did you also restart BitlBee? In an earlier comment you said you didn't want to, but AFAIK the only way to load an updated plugin is to restart.

**Comment by varazir** [comment]
> @varazir Did you also restart BitlBee? In an earlier comment you said you didn't want to, but AFAIK the only way to load an updated plugin is to restart.

Yes I reconnected, previous version crashed bitlbee when I tried to close the channel windows.

**Comment by dgw** [comment]
Does this happen for all channels, or just one in particular?

I see @sm00th already covered the meaning of this specific error code, but it looks like we haven't yet established how widespread the issue is.

**Comment by varazir** [comment]
All I have tested in and works fine from web gui
